### DISCUSSION TITLE
Figure and table caption has a full stop(.) between caption number and colon in PDF output

### DISCUSSION THREAD

**Discussion by yoonghm** [quarto-dev/quarto-cli#6834] [Q&A] [answered]
### Description

I have a qmd file wih a figure:

```
![Comparison of various databases](./img/Database_Compare.png){#fig-databases}
```

HTML output shows:
![image](https://github.com/quarto-dev/quarto-cli/assets/4215991/728522a1-7193-453c-8bcf-a7cc7224dee7)

However, PDF out shows:
![image](https://github.com/quarto-dev/quarto-cli/assets/4215991/c1ef6207-b411-40f2-8111-a2bb18a5de19)

How do I hide or remove the full stop (.) between caption number and colon?

**Comment by mcanouil** [comment]
Right now, I have no idea.

Please take the habit to provide a complete reproducible Quarto document and not just parts of one and screenshots. Thank you.

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

The end.
````
`````

**Comment by yoonghm** [comment]
I tried to create a simple document from project. The problem does not appear. Let me check out myself. Thank you for the pointer.

**Reply by yoonghm** [reply]
The full stop appeared once I have added an Appendix qmd document.
[Test.zip](https://github.com/quarto-dev/quarto-cli/files/12611672/Test.zip)

**Reply by mcanouil** [reply]
Could you share it as plain text as showed in my previous reply or as a Git repository?

**Comment by yoonghm** [comment]
**`_quarto.yml`**:

````yml
```yml
project:
  type: book

book:
  title: "Figure and table caption has a full stop(.) between caption number and colon in PDF output #6834"
  author: "Yoonghm"
  date: "15/September/2023"
  output-file: "6834"
  chapters:
    - index.qmd
  appendices:
    - File.Database.qmd

format:
  html:
    theme: cosmo
    page-layout: article
    grid:
      sidebar-width: 300px
      body-width: 900px
      margin-width: 300px
      gutter-width: 1.5rem
    number-sections: true
    number-depth: 1
  pdf:
    documentclass: scrbook
    classoption: twoside
    toc: true
    toc-depth: 3
    toc-title: Table of Contents
    lof: true
    lot: true
    geometry:
      - top=20mm
      - inner=20mm
      - outer=10mm
      - bottom=20mm
      - footskip = \dimexpr\headsep+\ht\strutbox\relax
    papersize: A4
    fontsize: 12pt
    keep-tex: true
    keep-md: true
    number-sections: true
    number-depth: 0
    include-in-header: 
       text: |
         \usepackage{fvextra}
         \DefineVerbatimEnvironment{Highlighting}{Verbatim}{breaklines,commandchars=\\\{\}}
    include-before-body:
      text: |
        \RecustomVerbatimEnvironment{verbatim}{Verbatim}{
          showspaces = false,
          showtabs = false,
          breaksymbolleft={},
          breaklines
          % Note: setting commandchars=\\\{\} here will cause an error 
        }
```
````

**`index.qmd`**:

````qmd
```qmd
# Introduction

## Popular SQL Databases

![Comparison of various databases](./img/Database_Compare.png){#fig-databases}
```
````

**`File.Database.qmd`**:
````qmd
```qmd
# Supported File and Database Types

Blah blah
```
````

**Reply by mcanouil** [reply]
Your example is overly complex for no obvious reason. Please simplify your example to only include what's needed to highlight your issue (remove every unnecessary options, such as the HTML format, `lof`, etc.).

Try to expand from the following to figure out which of your options causes the "dot" to appear:

````qmd
---
format:
  pdf:
    documentclass: scrbook
---

# Header 1

## Sub header 1

![Quarto Logo](https://quarto.org/quarto.png){#fig-logo}
````

**Reply by mcanouil** [reply]
It seems to be because of the appendices.

**Reply by mcanouil** [reply, marked as answer]
See <https://tex.stackexchange.com/questions/444129/dots-after-chapter-and-section-numbers-in-scrbook-when-using-chapters-in-appendi>.

TL;DR:

```yaml
classoption:
  - numbers=noenddot
```

**Reply by yoonghm** [reply]
Thanks!
